src/coreclr/tools/aot/ILCompiler.ReadyToRun/ObjectWriter/SectionBuilder.cs

```diff
@@ -317,6 +317,10 @@ public SectionBuilder(TargetDetails target)
                     _codePadding = 0x002A0005u;
                     break;
 
+                case TargetArchitecture.Riscv64:
+                    _codePadding = 0x002A0005u;
```

Comment:
Could you explain why it is `0x002A0005u`? In other archs, it is `break` instruction. Please clarify.

Reply:
Done